templates/solaredge.yaml

```diff
@@ -43,22 +43,16 @@ pred_bat:
   #  - number.givtcp_{geserial}_load_power
   soc_kw:
     - sensor.solaredge_b1_available_energy
-  #soc_percent:
-  #  - sensor.solaredge_b1_state_of_energy
   soc_max:
     - sensor.solaredge_b1_maximum_energy
   reserve:
-    - number.solaredge_i2_backup_reserve
```

Comment:
this block probably needs to just be removed, or commented with a "you can only enable this if you have a backup interface" note